dashboard/app/models/backpack.rb

```diff
@@ -11,7 +11,7 @@
 #
 # Indexes
 #
-#  index_backpacks_on_storage_app_id  (storage_app_id) UNIQUE
+#  index_backpacks_on_storage_app_id       (storage_app_id) UNIQUE
```

Comment:
This is an unrelated update to whitespace. I unintentionally deleted whitespace and it reappears on a new data migration. @molly-moen